backend/src/store/store.go

```diff
@@ -77,7 +77,7 @@ func (s *Store) StartVoting(roomID types.ID) {
 	}
 
 	room.StartVoting()
-
+	go room.AutoRevealIfTimeIsUp()
```

Comment:
Should we disable the input field during the session? It does not effect the current one and it could confuse users.